AeroBody/Body/knowledge.md

```diff
@@ -11,3 +11,8 @@ The study of how fluid moves. We use fluid dynamics to create objects which inte
 [Surfacing Course](surfacing_course.md)
 
 Solidworks surfaces has been used throughout the creation of Car 3's shell. Compared to solidworks features, surfaces gives us greater flexibility to create a smooth, uniform, and unique shell. Go through the surfaces course to learn the elements needed to design your own concept.
+
+## Recruitment
+[Recruitment SOP](https://github.com/Solar-Gators/docs/blob/master/AeroBody/Body/PDFs/SOP_01Recruitment.pdf)
```

Comment:
Can we link this like this `[Recruitment SOP](/AeroBody/Body/PDFs/SOP_01Recruitment.pdf)`